Documentation/install/upgrade.rst

```diff
@@ -33,14 +33,13 @@ avoids ErrImagePull errors during the rollout.
 
 .. code:: bash
 
-    helm template cilium \
+    helm install cilium/cilium \
```

Comment:
Shouldn't this receive the `|CHART_RELEASE|` treatment too? The point of this section is to prepare all of the nodes with the downloaded Cilium image to ensure a quick upgrade.

Reply:
Added